content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '.lastIndexOf()' 
+Description: 'Searches a string for a specified value, and returns the position of the match.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
+  - 'Method'
+  - 'Characters'
+CatalogContent:
+  - 'learn-Java'
+  - 'paths/computer-science'
+---
+
+The **`.lastIndexOf()`** method is a method that searches a string for a specified value, and returns the position of the match. It searches the string from the end to the beginning, and returns the index within the string of the last occurrence of the specified value, or -1 if the value is not found.

```

Comment:
```suggestion
The **`.lastIndexOf()`** method searches a string for a specified value and returns the position of the match. It searches the string, from the end to the beginning, and returns the index of the specified value. In other words, it returns the index of the last occurrence of the specified value. Otherwise if the value is not found, it returns -1.

```